partner_credit_limit/models/partner.py

```diff
@@ -92,3 +95,23 @@ def get_allowed_sale(self):
         for partner in self:
             partner.allowed_sale = not partner.credit_overloaded and \
                 not partner.overdue_credit
+
+    def _check_limit_credit(self):
+        """Method that will return a message when the partner has late payments and/or has exceeded the credit limit.
+        """
+        self.ensure_one()
+        if self.allowed_sale:
+            return False
+        credit_overloaded = self.credit_overloaded
+        overdue_credit = self.overdue_credit
+        msg = _('Can not confirm the %s because the Partner:\n') % (self.env.context.get('model_name', _('record')))
+        do_msg = _('Please: \n')
+
+        if overdue_credit:
+            msg += TAB_SPACE + _('• Has late payments.\n')
+            do_msg += TAB_SPACE + _('• Cover the late payments.\n')
+        if credit_overloaded:
+            msg += TAB_SPACE + _('• Has exceeded the credit limit.\n')
+            do_msg += TAB_SPACE + _('• Check credit limit.\n')
+            do_msg += TAB_SPACE*2 + _('Credit limit: %s') % self.credit_limit
```

Comment:
This is not so clear IMHO.

What do you think about using a method like the following?

```
def indent_meesage(msg, indent_level=1):
```